### PR TITLE
Update pull_request_template.md to include package name review template

### PR DESCRIPTION
**Pull request by ijnek**
Updates issue template

### CODE REVIEW

**Comment by ijnek** [comment]
@nuclearsandwich fyi

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -47,6 +47,21 @@ More info at https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidel
...
+## Package name
+
+package_name
+
+## Package source code
+
+http://sourcecode_url
+
+# Checks
+- [ ] Does the package naming comply with [REP-144](https://ros.org/reps/rep-0144.html) ?
```
Comment (audrow) [inline .github/pull_request_template.md:64]:
```suggestion
- [ ] Does the package naming comply with [REP-144](https://ros.org/reps/rep-0144.html)? 
```

**Review by audrow — COMMENTED** [review]
Seems okay to me. I'd like to get some other eyes on it before merging.

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]
Thanks for making the suggestion here! I brought this up in today's ROS core meeting and we're concerned with adding an additional tree to the "choose your own adventure" template. Do you think you could instead make the REP-144 requirement part of the DOC INDEX review, which already serves a similar purpose of registering a source repository to be indexed in rosdistro?

**Comment by ijnek** [comment]
Thanks for the review @audrow and @nuclearsandwich , i have addressed your feedback in #40518. Closing this PR since it is superceded.

**Pull request closed without merging**